From 449cd39af72fea4889864d92c6175b6f2c851350 Mon Sep 17 00:00:00 2001
From: David Mitchell <davem@iabyn.com>
Date: Wed, 25 Mar 2015 17:11:40 +0000
Subject: Perl_save_re_context(): re-indent after last commit

whitespace-only change.

(cherry picked from commit 4e0341d2ce817c9956f7f78e36bcaf8b764e18fc)

Bug: https://rt.perl.org/Ticket/Display.html?id=124109
Bug-Debian: https://bugs.debian.org/820328
Patch-Name: fixes/regcomp_whitespace_fix.diff
---
 regcomp.c | 28 ++++++++++++----------------
 1 file changed, 12 insertions(+), 16 deletions(-)

diff --git a/regcomp.c b/regcomp.c
index 7d1aa28c9..84af01134 100644
--- a/regcomp.c
+++ b/regcomp.c
@@ -16490,22 +16490,18 @@ Perl_save_re_context(pTHX)
     if (nparens == -1)
         nparens = 3;
 
-    {
-        {
-	    for (i = 1; i <= nparens; i++) {
-		char digits[TYPE_CHARS(long)];
-		const STRLEN len = my_snprintf(digits, sizeof(digits),
-                                               "%lu", (long)i);
-		GV *const *const gvp
-		    = (GV**)hv_fetch(PL_defstash, digits, len, 0);
-
-		if (gvp) {
-		    GV * const gv = *gvp;
-		    if (SvTYPE(gv) == SVt_PVGV && GvSV(gv))
-			save_scalar(gv);
-		}
-	    }
-	}
+    for (i = 1; i <= nparens; i++) {
+        char digits[TYPE_CHARS(long)];
+        const STRLEN len = my_snprintf(digits, sizeof(digits),
+                                       "%lu", (long)i);
+        GV *const *const gvp
+            = (GV**)hv_fetch(PL_defstash, digits, len, 0);
+
+        if (gvp) {
+            GV * const gv = *gvp;
+            if (SvTYPE(gv) == SVt_PVGV && GvSV(gv))
+                save_scalar(gv);
+        }
     }
 }
 #endif
